chronoreferencing-italian-slovenian-borderlands-anonymous.ipynb

```diff
@@ -13,7 +13,7 @@
     ]
```

Comment:
<p>to be removed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jdh-observer/6ig87tC5GKjQ/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='173'/>